central/pruning/pruning.go

```diff
@@ -430,10 +438,18 @@ func (g *garbageCollectorImpl) removeOrphanedProcessBaselines(deployments set.Fr
 // removeOrphanedPLOPs: cleans up ProcessListeningOnPort objects that are expired
 // or have a PodUid and belong to a deployment or pod that does not exist.
 func (g *garbageCollectorImpl) removeOrphanedPLOPs() {
-	prunedCount := postgres.PruneOrphanedPLOPs(pruningCtx, g.postgres, orphanWindow)
+	prunedCount := g.plops.PruneOrphanedPLOPs(pruningCtx, orphanWindow)
 
 	log.Infof("[PLOP pruning] Found %d orphaned process listening on port objects",
 		prunedCount)
+
+	prunedCount, err := g.plops.RemovePLOPsWithoutProcessIndicatorOrProcessInfo(pruningCtx)
+
```

Comment:
nit: remove new line

Reply:
I am not 100% clear which lines to remove, but I removed the ones that seemed to be the best to remove.